### PR TITLE
feat: get team schedule and populate teams

### PR DESCRIPTION
**Pull request by gabrield2731**
added cli functionality to get a schedule for a team and to create test hackers and teams

### CODE REVIEW

**Comment by gabrield2731** [comment]
might be some weird merge conflict not sure though

File: scripts/cli-util/get-team.ts [hunk 1]
```diff
@@ -65,7 +65,22 @@ export const handleGetTeam = async () => {
 };
 
 const getSchedule = async (team: TeamData) => {
-	console.log('IN PROGRESS');
+	const schedules: ScheduleData[] | null = await Schedule.find({
+		team: team._id,
+	});
+	schedules.forEach(schedule => {
+		console.log('Schedule:');
+		console.log(`Zoom: ${schedule.zoom}`);
```
Comment (zineanteoh) [inline scripts/cli-util/get-team.ts:73]: we no longer need zoom for schedule. ScheduleData model is really outdated. but we do need the table number (can be found in the team's document)

File: scripts/cli-util/populate-teams.ts [hunk 2]
```diff
@@ -1,17 +1,58 @@
...
+};
 
-	// TODO: gabe
+const createTeams = async () => {
+	for (let i = 0; i < 10; ++i) {
+		const teamName = `${faker.commerce.productAdjective()} ${faker.commerce.productAdjective()} ${faker.commerce.product()}`;
+		const team = new Team({
+			name: teamName,
+			members: [],
+			devpost: 'test devpost',
```
Comment (zineanteoh) [inline scripts/cli-util/populate-teams.ts:41]: generate a valid devpost link starting with https://devpost.com/{blablabla}

**Review by zineanteoh — CHANGES_REQUESTED** [review]
thanks @gabrield2731 . have you tested your code on our dev database?

File: scripts/cli-util/delete-collection.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ export const handleDeleteCollection = async () => {
 	}
 
 	if (del) {
-		await User.deleteMany({ userType: 'HACKER' });
+		await User.deleteMany({ userType: 'HACKER', email: { $ne: 'test-hacker@vandyhacks.dev' } });
```
Comment (zineanteoh) [inline scripts/cli-util/delete-collection.ts:72]:
```suggestion
		await User.deleteMany({ userType: 'HACKER', email: { $ne: 'testhacker@vandyhacks.dev' } });
```

**Review by zineanteoh — APPROVED** [review]
lgtm -- thanks @gabrield2731 !

**Pull request merged**